helpers/import.sh

```diff
@@ -46,7 +46,7 @@ Example of required Terraform configuration:
 
     module "forseti" {
       source = "terraform-google-modules/forseti/google"
-      version = "~> 4.2"
+      version = "~> 5.0"
```

Comment:
This is a significant change and should be highlighted in title and description.  So, that it's easy to find when people are just scanning the PR.

Reply:
@blueandgold PTAL above